src/runtime/go.mod

```diff
@@ -4,114 +4,129 @@ go 1.19
 
 require (
 	code.cloudfoundry.org/bytefmt v0.0.0-20211005130812-5bb3c17173e5
-	github.com/BurntSushi/toml v1.2.0
+	github.com/BurntSushi/toml v1.2.1
 	github.com/blang/semver v3.5.1+incompatible
 	github.com/blang/semver/v4 v4.0.0
-	github.com/containerd/cgroups v1.0.5-0.20220625035431-cf7417bca682
+	github.com/containerd/cgroups v1.1.0
 	github.com/containerd/console v1.0.3
-	github.com/containerd/containerd v1.6.8
+	github.com/containerd/containerd v1.7.2
```

Comment:
@beraldoleal I'd like to double-check this bump is ok.

As per [`versions.yaml`](https://github.com/kata-containers/kata-containers/blob/main/versions.yaml#L217) we support 3 versions of containerd: `version` (minimum?), `lts` and `active` , based on https://containerd.io/releases/#support-horizon

On commit 7319cff77a @fidencio introduced CI for `lts` and `active`.

Bumping runtime to v1.7.2 it seems will replace the `version` required.

Reply:
Always good to be safe! Thanks @wainersm.

The issue is that if we are moving away from gogo, then 1.7 should be the minimum.